components/mission_protection_system/tests/runner.py

```diff
@@ -172,12 +173,15 @@ def run(script, args):
     if not MPS_SOCKET:
         p = pexpect.spawn(MPS_BIN)
     else:
+        _ = subprocess.Popen([MPS_BIN])
+        time.sleep(0.1)
         sock = socket.socket(socket.AF_UNIX, socket.SOCK_STREAM)
         sock.connect(MPS_SOCKET)
         p = pexpect.fdpexpect.fdspawn(sock.fileno())
         # Reset the MPS to its initial state.
-        p.sendline('R')
-        try_expect(p, 'RESET')
+        # TODO need to coordinate with the socket here
```

Comment:
Do we need to resolve this TODO?

Reply:
It was resolved by restarting the executable every time, because we no longer use an external long-running program and instead the test runner launches MPS itself